modules/restore-regenerate-etcd-certs.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-regenerate-etcd-certs_{context}"]
+= Regenerating etcd certificates
+
+If the etcd certificates have expired for your master host, you must regenerate them before you can add the member back to the etcd cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+// * SSH access to the master host to add to the etcd cluster.
+// * The IP address of an existing active etcd member.
+
+// TODO: Figure out prereqs and assumptions. you've already removed it?
+
+.Procedure
+
+. TODO: Check whether the certs are expired or not. If they're valid, skip procedure
++
+// TODO: right now, the following steps are a straight copy/paste from restoring lost master hosts procedure (step 5)
+
+. Set up a temporary etcd certificate signer service on your master where you have restored etcd.
++
+// TODO: should this be run on one of the other healthy members?
+
+.. Access the original master, and log in to your cluster as a `cluster-admin` user using the following command.
++
+----
+[core@ip-10-0-143-125 ~]$ oc login https://localhost:6443
+Authentication required for https://localhost:6443 (openshift)
+Username: kubeadmin
+Password:
+Login successful.
+----
+
+.. Obtain the pull specification for the `kube-etcd-signer-server` image.
++
+----
+[core@ip-10-0-143-125 ~]$ export KUBE_ETCD_SIGNER_SERVER=$(sudo oc adm release info --image-for kube-etcd-signer-server --registry-config=/var/lib/kubelet/config.json)
+----
+
+.. Run the `tokenize-signer.sh` script.
++
+Be sure to pass in the `-E` flag to `sudo` so that environment variables are properly passed to the script.
++
+----
+[core@ip-10-0-143-125 ~]$ sudo -E /usr/local/bin/tokenize-signer.sh ip-10-0-143-125 <1>
+Populating template /usr/local/share/openshift-recovery/template/kube-etcd-cert-signer.yaml.template
+Populating template ./assets/tmp/kube-etcd-cert-signer.yaml.stage1
+Tokenized template now ready: ./assets/manifests/kube-etcd-cert-signer.yaml
+----
+<1> The host name of the original master you just restored, where the signer should be deployed.
```

Comment:
`The host name of the healthy master, where you are attempting to deploy the signer.`